helper-cli/src/main/kotlin/commands/SetLabelsCommand.kt

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2021 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.helper.commands
```

Comment:
Instead of this approach, have you considered doing a command which 

(1) extracts the labels to a yaml file
(2) set the labels from a yaml file

This way you can just open up a yaml file and edit the fields. I believe this is quicker than doing it via the command line.

Reply:
After discussing this it seems that as-is fits your use case better.